pdr_backend/aimodel/aimodel_plotter.py

```diff
@@ -249,39 +247,63 @@ def plot_aimodel_varimps(d: AimodelPlotdata):
     @arguments
       d -- AimodelPlotdata
     """
-    var_imps, errors = d.model.importance_per_var(include_stddev=True)
-    labels = d.colnames
-
-    var_imps = [100 * a for a in var_imps]
-    errors = [100 * a for a in errors]
-
-    df = pd.DataFrame(var_imps, columns=["importance"])
-    df["label"] = labels
-    df["erorrs"] = errors
-    df["low"] = df["importance"] - df["erorrs"] * 2
-    df["high"] = df["importance"] + df["erorrs"] * 2
-    df.sort_values(by=["importance"], inplace=True, ascending=True)
-
-    chart = (
-        alt.Chart(df)
-        .mark_bar()
-        .encode(
-            x=alt.X("importance", title="Relative importance (%)"),
-            y=alt.Y("label", title=None, sort="-x", axis=alt.Axis(labelLimit=200)),
+    imps_avg, imps_stddev = d.model.importance_per_var(include_stddev=True)
```

Comment:
was this breaking? i ask because the return vars changed, and i remember that block of code being here and it was removed. So, I guess you're returning the truncation?